mimic/rest/valkyrie_api.py

```diff
@@ -0,0 +1,49 @@
+# -*- test-case-name: mimic.test.test_valkyrie -*-
+"""
+API Mock for Valkyrie.
+"""
+
+from mimic.rest.mimicapp import MimicApp
+
+class ValkyrieApi(object):
+    """
+    Rest endpoints for the Valkyrie API.
+    """
+
+    app = MimicApp()
+
+    def __init__(self, core):
+        """
+        :param MimicCore core: The core to which the Valkyrie Api will be
+        communicating.
+        """
+        self.core = core
+
+    @app.route('/login', methods=['POST'])
+    def login(self, request):
+        """
+        Responds with response code 200 and returns an auth token
+        """
+        return self.core.valkyrie_store.create_token(request)
+
+    @app.route('/login_user', methods=['POST'])
+    def login_user(self, request):
+        """
+        Responds with response code 200 and returns an auth token
+        """
+        return self.core.valkyrie_store.create_token(request)
+
+    @app.route('/account/<string:account_number>/permissions/contacts/accounts/by_contact/<string:contact_id>/effective', methods=['GET'])
```

Comment:
If memory serves me correctly, you can specify <int:contact_id> and Klein will perform the appropriate type-conversion for you.